db/migrate/20160720151217_create_arbitration_rules.rb

```diff
@@ -0,0 +1,16 @@
+class CreateArbitrationRules < ActiveRecord::Migration[5.0]
+  def change
+    create_table :arbitration_rules do |t|
+      t.string :type
+      t.string :name
+      t.string :parameter
+      t.string :condition
+      t.string :value
+      t.string :action
```

Comment:
Sorry...I saw the definitions in the OP, but even the definitions were generic, e.g. `parameter: the parameter of the request to look at`.  I'm afraid something like the condition column will get confused with the Condition model.  Additionally, a lot of this feels like rehash of MiqExpression...would it make more sense to use that?